scripts/update_VGA.sh

```diff
@@ -51,8 +51,24 @@ else
     if [ -r /media/*/VBOXADDITIONS*/VBoxLinuxAdditions.run ]; then
       VGArun=/media/*/VBOXADDITIONS*/VBoxLinuxAdditions.run
     else
-        echo "CD still not found. Aborting!" >> /dev/stderr
-        exit
+        echo "CD still not found. Trying /dev/sr0 !" >> /dev/stderr
+
+        if [ ! -e /media/cdrom ]; then
```

Comment:
`-e` tests for exists, but arguably should test if it's a directory. However, I would skip the test and just do the `mkdir -p` (as the flag makes sure it won't do any harm if it exists)